docs/design/features/hybrid-globalization.md

```diff
@@ -198,3 +198,118 @@ Web API does not expose locale-sensitive endsWith/startsWith function. As a work
 
 - `IgnoreSymbols`
 Only comparisons that do not skip character types are allowed. E.g. `IgnoreSymbols` skips symbol-chars in comparison/indexing. All `CompareOptions` combinations that include `IgnoreSymbols` throw `PlatformNotSupportedException`.
+
+
+### OSX
+
+For OSX platforms we are using native apis instead of ICU data.
+
+**String comparison**
+
+Affected public APIs:
+- CompareInfo.Compare,
+- String.Compare,
+- String.Equals.
+
+The number of `CompareOptions` and `NSStringCompareOptions` combinations are limited. Originally supported combinations can be found [here for CompareOptions](https://learn.microsoft.com/dotnet/api/system.globalization.compareoptions) and [here for NSStringCompareOptions](https://developer.apple.com/documentation/foundation/nsstringcompareoptions).
+
+- `IgnoreSymbols` is not supported because there is no equivalent in native api. Throws `PlatformNotSupportedException`.
+
+- `IgnoreKanaType` is not supported because there is no equivalent in native api. Throws `PlatformNotSupportedException`.
+
+- `StringSort` is not supported because there is no equivalent in native api. Throws `PlatformNotSupportedException`.
```

Comment:
`StringSort` curently works like `None`, docs on that are out of date. You can check that 
https://learn.microsoft.com/en-us/dotnet/api/system.globalization.compareoptions?view=net-7.0#examples

does not give the result the docs promises and actually gives the result as it's expected from `None`. From this reason, Browser supports SortString and I think iOS should as well.

That's the actual resutlt on net7 Console App:, same with Blazor on net7 (that is using ICU4C):
```
Initially,
cant
bill's
coop
cannot
billet
can't
con
bills
co-op

After sorting without CompareOptions.StringSort:
bill's
billet
bills
can't
cannot
cant
co-op
con
coop

After sorting with CompareOptions.StringSort:
bill's
billet
bills
can't
cannot
cant
co-op
con
coop
```

Btw, do you know how we can change the official docs on the coreclr behavior?

Reply:
https://github.com/dotnet/dotnet-api-docs/issues/9014

Reply:
Does it mean in browser CompareOptions.StringSort is mapped to None? Maybe this is a bug on .net 7 ?
Not sure how to change official docs.

Reply:
Yes, it means that Browser's None and StringSort share the same logic. It's not only net 7, it's all "new" dotnets.

Reply:
Updated for OXS also.